Mastodon/Scene/Profile/ProfileViewController.swift

```diff
@@ -202,33 +199,38 @@ extension ProfileViewController {
             }
             .store(in: &disposeBag)
 
-        Publishers.CombineLatest4 (
-            viewModel.relationshipViewModel.$isSuspended,
+        // build items
+        Publishers.CombineLatest4(
+            viewModel.$relationship,
             profileHeaderViewController.viewModel.$isTitleViewDisplaying,
-            editingAndUpdatingPublisher.eraseToAnyPublisher(),
-            barButtonItemHiddenPublisher.eraseToAnyPublisher()
+            editingAndUpdatingPublisher,
+            barButtonItemHiddenPublisher
         )
         .receive(on: DispatchQueue.main)
-        .sink { [weak self] isSuspended, isTitleViewDisplaying, tuple1, tuple2 in
-            guard let self = self else { return }
+        .sink { [weak self] account, isTitleViewDisplaying, tuple1, tuple2 in
+            guard let self else { return }
             let (isEditing, _) = tuple1
             let (isMeBarButtonItemsHidden, isReplyBarButtonItemHidden, isMoreMenuBarButtonItemHidden) = tuple2
 
             var items: [UIBarButtonItem] = []
             defer {
-                self.navigationItem.rightBarButtonItems = !items.isEmpty ? items : nil
+                if items.isNotEmpty {
+                    self.navigationItem.rightBarButtonItems = items
+                } else {
+                    self.navigationItem.rightBarButtonItems = nil
+                }
             }
 
-            guard !isSuspended else {
+            if let suspended = self.viewModel.account.suspended, suspended == true {
```

Comment:
Maybe

```suggestion
            guard self.viewModel.account.suspended == true else {
```

to have it uniform with the other `guard`?